examples/deadline/All-In-AWS-Infrastructure-Basic/ts/README.md

```diff
@@ -32,7 +32,13 @@ These instructions assume that your working directory is `examples/deadline/All-
     ```
     aws secretsmanager create-secret --name <name> --secret-binary fileb://<path-to-zip-file>
     ```
-5.  The output from the previous step will contain the secret's ARN. Change the value of the `ublCertificatesSecretArn` variable in `bin/config.ts` to the secret's ARN:
+
+    ---
+
+    **Note:** The next two steps are for setting up usage based licensing and are optional. You may skip these if you do not need to use licenses for rendering.
```

Comment:
I think I missed this last time, but we could also skip the previous step as well right? We don't need to create a secret with our UBL certs since we aren't using them.

Reply:
Oh, whoops, I didn't notice that. Yes I'll update this, nice catch.